src/adapter/cloudflare-pages/handler.test.ts

```diff
@@ -2,30 +2,61 @@ import { getCookie } from '../../helper/cookie'
 import { Hono } from '../../hono'
 import { HTTPException } from '../../http-exception'
 import type { EventContext } from './handler'
-import { handle, handleMiddleware } from './handler'
+import { handle, handleMiddleware, serveStatic } from './handler'
 
 type Env = {
   Bindings: {
     TOKEN: string
-    eventContext: EventContext
+  }
+}
+
+function createEventContext(
+  context: Partial<EventContext<Env['Bindings']>>
+): EventContext<Env['Bindings']> {
+  return {
+    data: {},
+    env: {
+      ...context.env,
+      ASSETS: { fetch: vi.fn(), ...context.env?.ASSETS },
+      TOKEN: context.env?.TOKEN ?? 'HONOISCOOL',
+    },
+    functionPath: '_worker.js',
+    next: vi.fn(),
+    params: {},
+    passThroughOnException: vi.fn(),
+    request: new Request('http://localhost/api/foo'),
+    waitUntil: vi.fn(),
+    ...context,
```

Comment:
Creating an `EventContext` in each test was getting quite repetitive, so I added this function that turns a `Partial<EventContext>` into an `EventContext` with all the necessary properties.